test/async/mandos/promises/parent-child-async/one_async_call_success_cb.scen.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "one_async_call_success_cb",
+    "steps": [
+        {
+            "step": "setState",
+            "accounts": {
+                "address:user": {
+                    "nonce": "0",
+                    "balance": "0",
+                    "storage": {},
+                    "code": ""
+                },
+                "0x00000000000000000f0f706172656e7453432e2e2e2e2e2e2e2e2e2e2e2e2e2e": {
```

Comment:
Mass-replace "0x00000000000000000f0f706172656e7453432e2e2e2e2e2e2e2e2e2e2e2e2e2e" with "sc:parent" to make your life easier.

Reply:
We'll look into all of them as soon as `feat/promises-final` is merged into `master`.